site/docs/releases.md

```diff
@@ -51,44 +51,60 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
   <dependency>
     <groupId>org.apache.iceberg</groupId>
     <artifactId>iceberg-core</artifactId>
-    <version>0.10.0</version>
+    <version>0.11.0</version>
   </dependency>
   ...
 </dependencies>
 ```
 
-## 0.10.0 release notes
+## 0.11.0 release notes
 
 High-level features:
 
-* **Format v2 support** for building row-level operations (`MERGE INTO`) in processing engines
-    * Note: format v2 is not yet finalized and does not have a forward-compatibility guarantee
-* **Flink integration** for writing to Iceberg tables and reading from Iceberg tables (reading supports batch mode only)
-* **Hive integration** for reading from Iceberg tables, with filter pushdown (experimental; configuration may change)
+* **Core API** now supports partition spec and sort order evolution
+* **Spark 3** now supports the following SQL extensions:
+    * MERGE INTO (experimental)
+    * DELETE FROM (experimental)
+    * ALTER TABLE ... ADD/DROP PARTITION
+    * ALTER TABLE ... WRITE ORDERED BY
+    * invoke stored procedures using CALL
+* **Flink** now supports streaming reads, CDC writes (experimental), and filter pushdown
+* **AWS module** is added to support better integration with AWS, with AWS Glue catalog support and dedicated S3 FileIO implementation
+* **Nessie module** is added to support integration with project Nessie
 
 Important bug fixes:
 
-* [\#1706](https://github.com/apache/iceberg/pull/1706) fixes non-vectorized ORC reads in Spark that incorrectly skipped rows
-* [\#1536](https://github.com/apache/iceberg/pull/1536) fixes ORC conversion of `notIn` and `notEqual` to match null values
-* [\#1722](https://github.com/apache/iceberg/pull/1722) fixes `Expressions.notNull` returning an `isNull` predicate; API only, method was not used by processing engines
-* [\#1736](https://github.com/apache/iceberg/pull/1736) fixes `IllegalArgumentException` in vectorized Spark reads with negative decimal values
-* [\#1666](https://github.com/apache/iceberg/pull/1666) fixes file lengths returned by the ORC writer, using compressed size rather than uncompressed size
-* [\#1674](https://github.com/apache/iceberg/pull/1674) removes catalog expiration in HiveCatalogs
-* [\#1545](https://github.com/apache/iceberg/pull/1545) automatically refreshes tables in Spark when not caching table instances
+* [\#2091](https://github.com/apache/iceberg/pull/2091) fixes Parquet vectorized reads when column types are promoted
+* [\#1991](https://github.com/apache/iceberg/pull/1991) fixes Avro schema conversions to preserve field docs
+* [\#1981](https://github.com/apache/iceberg/pull/1981) fixes date and timestamp transforms
```

Comment:
I'd like to have better descriptions of the important bug fixes. Release notes should provide context for understanding what's in the release. In this case, date and timestamp transforms were producing incorrect values for negative dates and times. Being a bit more specific helps people understand that they are probably not affected. And if they are, they should click to the issue.